### PR TITLE
Fix browse accordion tracking

### PR DESCRIPTION
**Pull request by andysellick**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What
Fix accordion 'show all sections' link tracking on https://www.gov.uk/browse/benefits/manage-your-benefit

The options being passed to the component were incorrectly formatted. I'll write up a better guide for reference after this, but here's the basics.

- the show all sections link is created by JS in the `govuk-frontend` code, so we can't append attributes directly to it
- instead we pass `data_attributes_show_all` to the component, which is read by our JS as data attributes to be appended to the show all link once created
- the format of `data_attributes_show_all` must be:

```ruby
<%
value1 = {
  something: 1
}

value2 = {
  something: 2
}
%>

# in the call to the component

data_attributes_show_all: {
  "attribute-name": value1.to_json,
  "another-attribute": value2.to_json,
}
```

This will produce attributes on the show all link that looks like the following.

```
data-attribute-name="{"something":1}" data-another-attribute="{"something":2}"
```

Also remove passing of the components own JS module identifier to the accordion, as it doesn't need to be - it's explicitly added [in the accordion template](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_accordion.html.erb#L25).

## Why
Incorrectly formatted option was resulting in a data attribute of `[Object object]` being incorrectly appended to the link, which was then unable to be interpreted as JSON as expected by the accordion component JavaScript.

## Visual changes
None.

### CODE REVIEW

**Review by hannalaakso — COMMENTED** [review]
It looks good to me, I can see the dimension 114 being set to `0` correctly. It would be great to get a second review, maybe @patrickpatrickpatrick. And some docs about the formatting would be great too.

**Review by patrickpatrickpatrick — APPROVED** [review]
Looks good to me. Apologies for changing how the attributes were added in the analytics js but forgetting to update the mark-up in collections.

**Pull request merged**
